src/tracks/playabletrack/wavetrack/ui/WaveTrackShifter.cpp

```diff
@@ -153,15 +153,24 @@ class WaveTrackShifter final : public TrackShifter {
       return ok;
    }
 
-   bool Attach( Intervals intervals ) override
+   bool Attach( Intervals intervals, double offset ) override
    {
       for (auto &interval : intervals) {
          auto pData = static_cast<WaveTrack::IntervalData*>( interval.Extra() );
          auto pClip = pData->GetClip();
          if ( !mpTrack->AddClip( pClip ) )
             return false;
          mMigrated.insert( pClip.get() );
-         mMoving.emplace_back( std::move( interval ) );
+         if(offset == .0)
+            mMoving.emplace_back( std::move( interval ) );
+         else
+         {
```

Comment:
Were the side effects on interval caused by move in the first branch important?  This branch omits them

Reply:
I suppose not, now I see they belong to expiring `remover`